barretenberg/cpp/src/barretenberg/dsl/acir_format/witness_constant.cpp

```diff
@@ -16,12 +17,20 @@ bb::stdlib::cycle_group<Builder> to_grumpkin_point(const WitnessOrConstant<FF>&
                                                    const WitnessOrConstant<FF>& input_y,
                                                    const WitnessOrConstant<FF>& input_infinite,
                                                    bool has_valid_witness_assignments,
+                                                   bool use_g1,
```

Comment:
Could you add the description of the functions. Right now you have to look inside to understand what the purpose of this argument